_posts/cas/2017-07-05-cas-contribution-guide.md

```diff
@@ -108,6 +110,14 @@ In order to successfully finish this exercise you need:
 2. [IntelliJ IDEA](https://www.jetbrains.com/idea/download/), eclipse or NetBeans (Depending on the change, `vim` may be perfectly fine too)
 3. [Java (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
 
+## File Contributor License Agreements
```

Comment:
Thank you.

My views certainly mimic yours and I too favor the removal of all sources of friction :) As the saying goes in my alternative tongue, "whose beard do we have to trim" to get traction on that policy and move forward?